topic/timeseries/requirements.txt

```diff
@@ -0,0 +1,5 @@
+# Real
+pycaret>=3.0
+mlflow
```

Comment:
Is this project relevant to be demonstrated together with mlflow-cratedb?

Reply:
For now, that's not the case. However, this notebook can further develop to cover some other topics.

Reply:
Does the notebook use MLflow at all? I can't see it in the code cells, but it might be an oversight.